routers/common/repo.go

```diff
@@ -89,11 +89,13 @@ func ServeData(ctx *context.Context, name string, size int64, reader io.Reader)
 		if (st.IsImage() || st.IsPDF()) && (setting.UI.SVG.Enabled || !st.IsSvgImage()) {
 			ctx.Resp.Header().Set("Content-Disposition", fmt.Sprintf(`inline; filename="%s"`, name))
 			if st.IsSvgImage() || st.IsPDF() {
-				ctx.Resp.Header().Set("Content-Security-Policy", "default-src 'none'; style-src 'unsafe-inline'; sandbox")
+				if !st.IsPDF() {
+					ctx.Resp.Header().Set("Content-Security-Policy", "default-src 'none'; style-src 'unsafe-inline'; sandbox")
+				}
```

Comment:
I think this was added here for security reasons otherwise JavaScript in pdf would be executed

Reply:
> You may as well remove the `|| st.IsPDF()` above for the same effect.

yes, modified

Reply:
> I think this was added here for security reasons otherwise JavaScript in pdf would be executed

With or without policy javascript will be executed

![image](https://user-images.githubusercontent.com/1255041/180581252-125cce88-3538-4e17-8dd0-b73c134ee3ba.png)